File: auto-docs/executables/julia/basic_heatmap.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

data = [
  [
    "z" => [1 20 30; 20 1 60; 30 60 1], 
    "type" => "heatmap"
  ]
]
response = Plotly.plot(data, ["filename" => "basic-heatmap", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/matlab/basic_heatmap.m
signin('TestBot', 'r1neazxo9w')
data = {...
  struct(...
    'z', [1, 20, 30; 20, 1, 60; 30, 60, 1], ...
    'type', 'heatmap')...
};
response = plotly(data, struct('auto_open', false, 'fileopt', 'overwrite', 'filename', 'basic-heatmap'));
plot_url = response.url